docker/launch_backend_service.sh

```
@@ -96,7 +96,13 @@ run_server(){
     local retry_count=0
     while ! $STOP && [ $retry_count -lt $MAX_RETRIES ]; do
         echo "Starting ragflow_server.py (Attempt $((retry_count+1)))"
-        $PY api/ragflow_server.py
+        if [[ "$*" == *"--debug"* ]]; then
+            echo "Debug mode is ACTIVE"
+            DEBUG_FLAGS=--debug
+            # Change this value to set a different debugpy listening port
+            export RAGFLOW_DEBUGPY_LISTEN=5678
+        fi
+        $PY api/ragflow_server.py $DEBUG_FLAGS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)